NetKet/Hamiltonian/MatrixWrapper/direct_matrix_wrapper.hpp

```diff
@@ -53,9 +59,9 @@ class DirectMatrixWrapper : public AbstractMatrixWrapper<WfType> {
       for (size_t k = 0; k < connectors.size(); ++k) {
         auto vk = v;
```

Comment:
Most likely this and the following line are quite slow... not sure how to improve on this without losing generality